src/components/card/__tests__/card-test-cases.js

```diff
@@ -35,4 +36,16 @@ testCases.noImage = {
   }
 };
 
+testCases.withLevelAndLanguage = {
+  component: Card,
+  description: 'Card with level and language',
+  props: {
+    title: 'This is a full-length title',
+    path: 'url',
+    description: "Here's a reasonable length for a description.",
+    level: <LevelIndicator level={2} />,
```

Comment:
This is one possibility for the interface; another would be to pass the level number directly. Curious why you chose to pass a React element instead of the level number? Do you think we may want to pass some other element in the future?

Reply:
Yea it's probably safe to just pass the level number.